src/server.ts

```diff
@@ -23,7 +23,7 @@ export function createServer (axios: AxiosStatic, redisClient: RedisClientWrappe
 
     try {
       const badge = await getBadgeImage(axios, redisClient, subject || DEFAULT_SUBJECT, lastVersion, color || DEFAULT_COLOR, format, style);
-      res.set('Cache-Control', 'public, max-age=43200'); // 12 hours
+      res.set('Cache-Control', 'no-cache');
```

Comment:
I'm not sure if this is a proper solution, right now our instance on Heroku can be easily overloaded with too many requests. Having 12h cache was a good compromise. And the issues with GH was mostly because GH cached version of a badge without `Cache-Control` defined. Dropping the badge from the GH cache can easily resolve the problem and then 12h period starts working.